cli/dts/lib.deno.ns.d.ts

```diff
@@ -185,7 +185,7 @@ declare namespace Deno {
      * console.log(Deno.env.get("MADE_UP_VAR"));  // outputs "Undefined"
      * ```
      * Requires `allow-env` permission. */
-    get(key: string): string | undefined;
+    get(key: string): string | null;
```

Comment:
I'm not sure we want to change declaration files - this could potentially be a breaking change. @kitsonk WDYT?

Reply:
I agree - this has to be reverted. This is a public API change to a stable API.

Reply:
Beyond the typing conversation, which is important, we should also reach a consensus on what should be serialized as `null` or `undefined` by `serde_v8`. Right now `serde_v8` serializes both `None` and `()` to `null`, which `serde_json::Value` also did, since `undefined` is not a JSON value, only `null` exists in JSON.

To return `undefined` from these ops we were doing hackish (IMO) workarounds, like returning a 0-or-1 sized arrays in the rust-op and then `[0]` in JS to the value or `undefined`.

With `serde_v8` we have the opportunity to change this behaviour and provide a mechanism for returning `undefined`, by mapping to it from `None` or `()`

I personally would keep everything as `null`, but I just want you to know that the option to change this behaviour is there and it would be best to decide on it before shipping `1.9`.

Reply:
you could keep the ops with null and do the null->undefined in js
(then this could potentially be changed/standardized all to nulls in 2.0)

Reply:
> you could keep the ops with null and do the null->undefined in js
> (then this could potentially be changed/standardized all to nulls in 2.0)

Yep, that's precisely what I suggested over in the discord chat about this (in `#dev-core`)

Reply:
These changes have been reverted and I've mapped `null`s to `undefined`.